query_classifier/qc_mysqlembedded/qc_mysqlembedded.cc

```diff
@@ -407,7 +407,7 @@ inline const char* qcme_string_get(const char* s)
     return s;
 }
 
-#define QC_CF_IMPLICIT_COMMIT_BEGIN CF_IMPLICT_COMMIT_BEGIN
+#define QC_CF_IMPLICIT_COMMIT_BEGIN CF_IMPLICIT_COMMIT_BEGIN
```

Comment:
The only current instance of this token is in a comment:
https://github.com/MariaDB/server/blob/799f75953f5b362736a2992fd5d92a14db94bb20/sql/sql_class.h#L7273

The live token is:
https://github.com/MariaDB/server/blob/799f75953f5b362736a2992fd5d92a14db94bb20/sql/sql_class.h#L7260